source/AI.cpp

```diff
@@ -2297,8 +2291,6 @@ void AI::DoAppeasing(const shared_ptr<Ship> &ship, double *threshold) const
 			toDump -= dumped;
 		}
 
-	// Merciful ships will let this one be.
-	ship->SetFleeing();
```

Comment:
I removed this because ships low on health already trigger the fleeing code from above.

Reply:
Its actually not the same threshold?
This one is based on weird stuff like 0.5 shields + hull

Reply:
Well I think this makes sense still, merciful doesnt necessarily mean only let fleeing flee, they would also just let cargo droppers alone?